### PR TITLE
(needs review) Don't give others your problems

### PR DESCRIPTION
**Pull request by m4sterbunny**
Requires further development as blog article
Requires proofing

### CODE REVIEW

File: content/blog/drafts/dinis/_transcribed/dont-give-others-your-problems.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
+ +editor: Harrie Bickle (TW)
+ +notes:
+ +  - not proofed/edited. Written on the fly in response to trigger article.
+ +---
+ +
+ +
+ ##Whose problem is this anyway?
+ +
+ #A problem shared is a problem halved
+ +There is something comforting about sharing an issue. Simply telling the tale to an understanding ear can somehow expand the cognitive space your own brain assigned to the problem. This can catalyse the formulation of the solution without your listener offering any opinions or insights themselves.
```
Comment (DinisCruz) [inline content/blog/drafts/dinis/_transcribed/dont-give-others-your-problems.md]: The problem with this version is that the core idea of "do not give others your problems." was lost

Comment (DinisCruz) [reply content/blog/drafts/dinis/_transcribed/dont-give-others-your-problems.md]: can you try to bring it back

**Comment by DinisCruz** [comment]
@m4sterbunny also please add to the title a good reference to what the change is all about

**Comment by m4sterbunny** [comment]
Dinis' version edited in place now

**Pull request closed without merging**